src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -101,6 +103,25 @@ void FaceCoordinatesCompute<Dim, Frame, Centered>::function(
 #pragma GCC diagnostic pop
 #endif  // defined(__GNUC__) && !defined(__clang__)
 
+template <size_t Dim>
+void PunctureFieldCompute<Dim>::function(
+    const gsl::not_null<return_type*> result,
+    const std::optional<tnsr::I<DataVector, Dim, Frame::Inertial>>&
+        inertial_face_coords,
+    const ::ExcisionSphere<Dim>& excision_sphere, const double time,
+    const size_t expansion_order) {
+  if (inertial_face_coords.has_value()) {
+    if (not result->has_value()) {
+      result->emplace(get<0>(inertial_face_coords.value()).size());
+    }
+    puncture_field(
+        make_not_null(&(result->value())), inertial_face_coords.value(), time,
+        get(magnitude(excision_sphere.center())), 1., expansion_order);
```

Comment:
Do you always assume the BH has mass 1 and also that the scalar charge is 1? Both should probably be quite prominently documented somewhere (maybe they already are). Or maybe put options for them in the input file and in places where you (have to) assume both are 1 you assert? If you want to relax this assumption you'd have to go through all of your code and find the places where you hardcoded literal ones, that can be quite annoying and hard to debug. Up to you.

Reply:
Yes, we had always assumed so until now, I never tried anything else. The charge is definitely 1, that is hard-coded. For the mass, I will probably add an OptionTag once I made sure it works for different masses.